readme.md

```diff
@@ -1,32 +1,40 @@
-# Lighthouse  [![Build Status](https://travis-ci.org/GoogleChrome/lighthouse.svg?branch=master)](https://travis-ci.org/GoogleChrome/lighthouse) [![Coverage Status](https://coveralls.io/repos/github/GoogleChrome/lighthouse/badge.svg?branch=master)](https://coveralls.io/github/GoogleChrome/lighthouse?branch=master)
+# Lighthouse  [![Build Status](https://travis-ci.org/GoogleChrome/lighthouse.svg?branch=master)](https://travis-ci.org/GoogleChrome/lighthouse) [![Coverage Status](https://coveralls.io/repos/github/GoogleChrome/lighthouse/badge.svg?branch=master)](https://coveralls.io/github/GoogleChrome/lighthouse?branch=master) [![NPM lighthouse package](https://img.shields.io/npm/v/lighthouse.svg)](https://npmjs.org/package/lighthouse)
 
-> Lighthouse analyzes web apps and web pages, collecting modern performance metrics and insights on developer best practices.
-
-**Lighthouse requires Chrome 56 or later.**
+> [Lighthouse](https://developers.google.com/web/tools/lighthouse/) analyzes web apps and web pages, collecting modern performance metrics and insights on developer best practices.
```

Comment:
> meh. We have the link at the top of the repo and sprinkled elsewhere.

should we just drop it from here then? With the link, the introductory sentence kind of undercuts the rest of the readme by saying "here's a place to go if you want to learn more about Lighthouse" rather than just reading the already-open readme :)